### DISCUSSION TITLE
Frame advance mod (i.e. osu! TAS)

### DISCUSSION THREAD

**Discussion by ramjam18** [ppy/osu#22157] [Ideas]
Would it be possible to add a mod where you assign two new keys, where one advances the gameplay by one frame, and the other will let it continue as normal for as long as it is held down?

Could open up a whole new realm of creativity: replay art!

PS: I realize that 'frame' might be a vague unit here... actually, insert whichever unit would be more suitable (milliseconds, ticks, idk).

**Comment by peppy** [comment]
Definitely an interesting idea!
